code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -181,7 +181,8 @@
 	return 1
 
 /obj/item/cardboard_cutout/setDir(newdir)
-	dir = SOUTH
+	newdir = SOUTH
+	return ..()
```

Comment:
I think you need `return ..(SOUTH)` here.
newdir only var in this proc and it dont put it nowhere.
or place `return ..(newdir)`

Reply:
it works fine as is.

Reply:
Really

Reply:
..() automatically forwards arguments

Reply:
I think the point is that this proc is overriding the argument by always setting SOUTH, but parent calls will inherit the now-wrong argument instead. I don't know if it has any potential to cause issues, but it should indeed be ..(SOUTH) in my opinion